vendor/github.com/elastic/gosigar/sigar_linux_common.go

```diff
@@ -282,6 +282,61 @@ func (self *ProcTime) Get(pid int) error {
 	return nil
 }
 
+func (self *ProcIO) Get(pid int) error {
+	// read proc io statistics from /proc/<pid>/io
```

Comment:
ooh, the procfs has a better way to get stati stics.
Can I use directly proc.NewProc methods?
like:
+func (self *ProcIO) Get(pid int) error {
+       // read proc io statistics from /proc/<pid>/io
+       p, err := proc.NewProc(pid)
+       if err != nil {
+               return err
+       }
+
+       count, err := p.NewIO()
+       if err != nil {
+               return err
+       }
+       self.ReadChar = count.RChar
+       self.WriteChar = count.WChar
+       self.ReadBytes = count.ReadBytes
+       self.WriteBytes = count.WriteBytes
+       return nil
+}